playwright.config.ts

```diff
@@ -21,16 +21,16 @@ const config: PlaywrightTestConfig = {
      */
     timeout: 15000,
   },
-  /* We don't want to run tests in parallel because we have the same backend, tests may conflict with eachother */
-  fullyParallel: false,
+  /* Default to run each suite in parallel, suites and optional opt out */
+  fullyParallel: true,
   /* Fail the build on CI if you accidentally left test.only in the source code. */
   forbidOnly: !!process.env.CI,
   /* Global setup file for initializing before all other tests */
   globalSetup: require.resolve('./tests/globalSetup.ts'),
   /* Retry on CI only */
   retries: process.env.CI ? 1 : 0,
-  /* Only have one worker since we don't want tests running in parallel, trampling over each other on the backend */
-  workers: 1,
+  /* Default to 50% of cores, don't want too many as core or web will become bottleneck */
+  workers: undefined,
```

Comment:
GitHub actions by default has 4 cores: https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories
Did you try defaulting to 100% in CI to see how it does?